/* Copyright (C) 2025 Interactive Brokers LLC. All rights reserved. This code is subject to the terms
 * and conditions of the IB API Non-Commercial License or the IB API Commercial License, as applicable. */

syntax = "proto3";

package protobuf;

import "TickAttribBidAsk.proto";

option java_outer_classname = "HistoricalTickBidAskProto";
option java_package = "com.ib.client.protobuf";
option csharp_namespace = "IBApi.protobuf";

message HistoricalTickBidAsk {
    optional int64 time = 1;
	optional TickAttribBidAsk tickAttribBidAsk = 2;
    optional double priceBid = 3;
    optional double priceAsk = 4;
    optional string sizeBid = 5;
    optional string sizeAsk = 6;
}
